tests/kafkatest/tests/client/compression_test.py

```diff
@@ -53,7 +53,7 @@ def min_cluster_size(self):
         # Override this since we're adding services outside of the constructor
         return super(CompressionTest, self).min_cluster_size() + self.num_producers + self.num_consumers
 
-    @cluster(num_nodes=7)
+    @cluster(num_nodes=8)
     @parametrize(compression_types=["snappy","gzip","lz4","zstd","none"])
```

Comment:
nit: there's a comment below that still refers to `num_producers=4`. Would be nice to figure find a way to make the dependence on the parameterization explicit. I wonder if we could make the list of compression types a static value and refer to its size when defining `num_producers` and `num_nodes`? Not a requirement, but if it's easy to do.